react/components/NumericStepper/README.md

```diff
@@ -127,6 +129,25 @@ initialState = {
       onChange={event => setState({ value4: event.value })}
     />
   </div>
+  <div className="mb5">
+    <NumericStepper
+      label="Disabled"
+      minValue={1}
+      disabled
+      value={state.value5}
+      onChange={event => setState({ value5: event.value })}
+    />
+  </div>
+  <div className="mb5">
+    <NumericStepper
```

Comment:
The lean version though disabled still have a hover effect. This has to be removed.

Reply:
Nice. I will do it.